### DISCUSSION TITLE
Was there a problem with Vercel service today?

### DISCUSSION THREAD

**Discussion by dawoon-coffeechat** [vercel/community#6913] [Feature Question] [answered]
### Summary

I was temporarily unable to access the service. What kind of problem did I encounter?

### Steps to Reproduce

Was there a problem with Vercel service today?

**Comment by swarnava** [comment]
Hey, what is the error you are receiving ? We don't have any widespread issue reported.

**Reply by noah-koala** [reply]
For the icn1 Seoul region there was ~

**Reply by dawoon-coffeechat** [reply]
I was unable to access the service for about 20 minutes.

No specific error occurred and it just didn't go beyond the white screen.

I wanted to find out more about the problem, but the problem was solved so I couldn't confirm it

**Reply by JISU-Y** [reply]
I got the same issue here. I'm in Korea too.

There wasn't any message that indicates an error. Just got an empty page for about 20 minutes.

From what I've noticed, It only shows an empty page when its page has a server side logic.

**Comment by swarnava** [comment, marked as answer]
We had received similar reports from customers located in Korea of a transient issue with Server Side Rendering pages being blank in the browser.

The team has confirmed this was caused by an update on our end in the icn1 region (Seoul) and a redeploy should fix the problem. Sorry about the inconvenience.

**Reply by dawoon-coffeechat** [reply]
ok thank you

We need to quickly identify these problems and take action.

There is no update on this issue in vercel-status. Please check.
